crates/aptos/src/common/supra/mod.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright © Aptos Foundation
```

Comment:
Should be `Entropy Foundation`

Reply:
done here https://github.com/Entropy-Foundation/aptos-core/pull/24/commits/0d32df201f21c10980fa175d2f60cea5c443df5d